backend/Techfesia2019/Techfesia2019/public_settings.py

```diff
@@ -27,3 +27,21 @@
 PUBLIC_ID_LENGTH = 10
 
 FIREBASE_CREDENTIALS_PATH = os.path.join(BASE_DIR, "Techfesia2019", "fake_creds.json")
+
+PAYTM_PRODUCTION = False
+
+PAYTM_MERCHANT_ID = 'JlLJuL45031399299246'
+
+PAYTM_SECRET_KEY = 's_yo0x1u_bow5toi'

```

Comment:
Don't commit these things here. Put in secret_settings.py.
Only put fake data in public_settings

Reply:
this is test data for test payments, real key is secure with me.